llvm/test/CodeGen/AMDGPU/machinelicm-crash.mir

```diff
@@ -0,0 +1,77 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1100 -run-pass=early-machinelicm %s | FileCheck %s
```

Comment:
More specific test file / function name, mention the issue number

Reply:
Done.